app/helpers/outcomes_helper.rb

```diff
@@ -1,6 +1,6 @@
 module OutcomesHelper
   def outcome_card_class(outcome)
-    if outcome.outcome_coverages.present?
+    if outcome.unarchived_outcome_coverages.present?
```

Comment:
What do you think about making this a "feature" of the `Outcome#outcome_coverages` association, just as we did with `Coverage#outcome_coverages`?

I think as written here, it's going to cause an N+1 query because each outcome in that view is going to filter the outcome coverages.